src/assets/styles.scss

```diff
@@ -39,7 +39,6 @@ $carousel-indicator-active-bg: $navbar-inverse-bg;
 $carousel-control-width: 5%;
 
 @import "~bootstrap/scss/bootstrap.scss";
-@import '~bootstrap-vue/src/index.scss';
```

Comment:
Why did you remove this import?

Reply:
This happened unintentionally :( I'll replace it now